pom.xml

```diff
@@ -20,47 +20,47 @@
     <repositories>
         <repository>
             <id>papermc</id>
-            <url>https://papermc.io/repo/repository/maven-public/</url>
-        </repository>
-        <repository>
-            <id>minecraft-repo</id>
-            <url>https://libraries.minecraft.net/</url>
+            <url>https://repo.papermc.io/repository/maven-public/</url>
         </repository>
         <repository>
             <id>minebench-repo</id>
             <url>https://repo.minebench.de/</url>
         </repository>
+        <repository>
+            <id>jitpack.io</id>
+            <url>https://jitpack.io</url>
```

Comment:
Please do not use JitPack. They cause lots of issues in the build process and can't produce consistent dependencies. In the past I worked around this by deploying OpenInv to my own repo and I would like to keep it that way. (Or alternatively use a version properly deployed to an official repo of course)

Reply:
Sadly that's the only current way to catch the official openInv versions. If you wish you can update your Version.
I certainly can't.

Reply:
4.4.3 (as well as 4.4.4-SNAPSHOT) are now part of my repo: https://repo.minebench.de/com/lishid/openinvapi/4.4.3/ so you should be able to just adjust the version of the current dependency without adding jitpack.

Reply:
The core not the api. But it's fine you did build the plugin itself as well ^^

Reply:
Well all artifacts that the project builds are in the repo, if you think the API is now enough (it wasn't in the past) then you can use that too I guess.

Reply:
You did misunderstand me.
The API isn't enough. However, you have sent the link for the API not the core ^^

Reply:
Aah, right, makes sense 😅